doc/developers-guide/mod_muc_light_developers_guide.md

```diff
@@ -12,7 +12,7 @@ All source files can be found in `apps/ejabberd/src`.
   Main module. 
   It implements `gen_mod` behaviour. 
   It subscribes to some essential hooks and exports several functions, mostly callbacks. 
-  It handles integration with `mod_mam` and Disco Service. 
+  It handles integration with `mod_disco`, `mod_privacy` and `mod_roster`.
```

Comment:
Not MAM anymore?

Reply:
There is no direct integration, as the MAM lookup requests are forwarded to `mod_muc_iq` module, which then may integrate with MAM, if enabled. Message archiving is being done via hooks that are not MAM-exclusive.